gateway-service-impl/src/main/java/org/hypertrace/gateway/service/explore/TimeAggregationsWithGroupByRequestHandler.java

```diff
@@ -107,9 +111,24 @@ private ExploreRequest buildGroupByRequest(ExploreRequest originalRequest) {
         .getTimeAggregationList()
         .forEach(timeAggregation -> requestBuilder.addSelection(timeAggregation.getAggregation()));
 
+    // remove interval based order by expression(if present) from order by expressions in the
+    // groupBy
+    // request being generated
```

Comment:
```suggestion
    // remove interval based order by expression(if present) from order by expressions in the
    // groupBy request being generated
```

Reply:
Can we add the reason to remove the interval based order by in case of group by, as a comment?

Reply:
Added